crates/pop-cli/README.md

```diff
@@ -0,0 +1,257 @@
+# Pop CLI
+
+<img src=".icons/logo.jpeg"></img>
+
+An all-in-one tool for Polkadot development.
+
+## Install
+
+You can install Pop CLI as follows:
+
+```shell
+cargo install --locked --git https://github.com/r0gue-io/pop-cli
+```
+
+> :information_source: A [crates.io](https://crates.io/crates/pop-cli) version will be available soon!
```

Comment:
With this separation, when does having `pop-cli` as a crate make sense? Rather shouldn't it be for `pop-parachain` and `pop-contracts`?

Reply:
Publishing the cli in the crate makes sense. It is recommended to publish it, users of Rust might install it from there. 
Examples: 
- https://crates.io/crates/subxt-cli
- https://crates.io/crates/cargo-contract

Reply:
Added the info message in crates `pop-parachain` and `pop-contracts `too